userguide/aws/tutorials/single-tier.adoc

```diff
@@ -0,0 +1,73 @@
+
+== Step 3: Set Up EC2 Configuration
+
+The next page will give several configuration options.
+
+image::../../assets/aws/single-ami-configure.png[align=center]
+
+Be sure to select the correct region for your account. Accept the other defaults, and click *Continue to Launch*.
+
+The next page will give you several options for the launch method. Choose *Launch through EC2*.
+
+image::../../assets/aws/single-ami-launch.png[align=center]
+
+All other options will disappear. Click *Launch*.
+
+== Step 4: Create a Key Pair
+
+The next page is headed *Launch an Instance*. Scroll down to the middle of this page, to the box titled *Key Pair (login)* Here you are asked to choose or create a *key pair*.
+
+image::../../assets/aws/single-tier-keypair.png[align=center]
+
+We will create a new key pair. Click *Create Key Pair* to display a pane to be used for the creation. Give the key pair a simple, easy to remember name such as `My-Key-Pair`. Do not change the other options on this pane. Then click *Download Key Pair*. *Important:* You *must* save the private key that will be generated in this step. If you lose this key, you will *not* be able to login to your instance. In addition, you must set the protection for this file so it is readable by the owner only.
+
+== Step 5: Launch Your Instance
+
+When the key pair is generated, click *Launch Instances*. You should see the screen below:
+
+image::../../assets/aws/single-tier-launching.png[align=center]
+
+Your instance is finally launching! To follow what is happening on the EC2 Dashboard, scroll all the way down to the bottom, and click *View Instance* at the bottom right. This will take you to the *Instances* screen which is part of the EC2 Dashboard.
+
+image::../../assets/aws/single-ami-instances.png[align=center]
+
+
+In a short time, the *Instance State* for your instance should indicate *Running*. Select the checkbox to the left of your instance ID. An information pane should open below with details about your instance.
+
+== Step 6: Testing
+
+You can now try to login to Kaui from your browser using the URL https://INSTANCE_IP:443, where INSTANCE_IP is the IPV4 address for your instance, given on your dashboard as *Public IPV4 Address*. If all goes well, this should display the Kaui login screen. The browser may complain that your connection is not secure, but it should offer you a way to bypass the problem.
+
+For an introduction to Kaui, see our https://docs.killbill.io/latest/userguide_kaui.html[Kaui Guide]. The default credentials are: `admin` / `{EC2 instance ID}`. The first few requests might be a bit slow as Kill Bill initializes itself.
+
+In addition, you can visit the Kill Bill server using the URL https://INSTANCE_IP:8443. This provides access to certain detailed reports that may be needed for maintenance, including metrics, event logs, and the Swagger API pages.
+
+
```

Comment:
Yes, I suggest that "updates" and "troubleshooting" should be separate, since they are about maintenance, not installation. Maybe "configuration" should still be included here.